### PR TITLE
HTML: Run a history navigation test in a popup to report a result

### PR DESCRIPTION
**Pull request by zcorpan**
Part of #11269.

### CODE REVIEW

File: html/browsers/browsing-the-web/history-traversal/window-name-after-cross-origin-main-frame-navigation.sub.html [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
     <script>
-        document.location = "support/window-name-navigation.sub.html?hostname={{domains[www1]}}&shouldhavename=false&sendmessage=false";
+    function popup() {
+      window.popupWin = window.open('support/window-name-after-cross-origin-main-frame-navigation-popup.sub.html', '_blank');
+    }
+    async_test(t => {
+      t.add_cleanup(() => {
+        popupWin.close();
+      })
+      document.getElementById('button').click();
```
Comment (foolip) [inline html/browsers/browsing-the-web/history-traversal/window-name-after-cross-origin-main-frame-navigation.sub.html]: Why the indirection via a button here? Popup blocking will still be in effect, so just inline all the code here?

**Review by foolip — APPROVED** [review]
LGTM % button indirection

File: html/browsers/browsing-the-web/history-traversal/window-name-after-cross-origin-main-frame-navigation.sub.html (hunk 1 above)
Comment (zcorpan) [reply html/browsers/browsing-the-web/history-traversal/window-name-after-cross-origin-main-frame-navigation.sub.html]: Just to make it slightly more convenient when running the test manually.

Comment (foolip) [reply html/browsers/browsing-the-web/history-traversal/window-name-after-cross-origin-main-frame-navigation.sub.html]: Ah, OK

**Pull request merged**